examples/edge_intelligence/windmill_graphite_sklearn.py

```diff
@@ -55,72 +55,72 @@
 
 
 def read_cpu_procs():
-    return cpu_stat.procs_running()
+	return cpu_stat.procs_running()
 
 
 def read_cpu_utilization(sample_duration_sec=1):
-    cpu_pcts = cpu_stat.cpu_percents(sample_duration_sec)
-    return round((100 - cpu_pcts['idle']), 2)
+	cpu_pcts = cpu_stat.cpu_percents(sample_duration_sec)
+	return round((100 - cpu_pcts['idle']), 2)
 
 
 def read_mem_free():
-    total_mem = round(mem_stat.mem_stats()[1], 4)
-    free_mem = round(mem_stat.mem_stats()[3], 4)
-    mem_free_percent = ((total_mem - free_mem) / total_mem) * 100
-    return round(mem_free_percent, 2)
+	total_mem = round(mem_stat.mem_stats()[1], 4)
+	free_mem = round(mem_stat.mem_stats()[3], 4)
+	mem_free_percent = ((total_mem - free_mem) / total_mem) * 100
+	return round(mem_free_percent, 2)
 
 
 def get_ambient_temperature(sensor_tag_collector):
-    return sensor_tag_collector.get_temperature()[0]
+	return sensor_tag_collector.get_temperature()[0]
 
 
 def get_relative_humidity(sensor_tag_collector):
-    return sensor_tag_collector.get_humidity()[1]
+	return sensor_tag_collector.get_humidity()[1]
 
 
 def get_pressure(sensor_tag_collector):
-    # 1 millibar = 100 Pascal
-    return sensor_tag_collector.get_barometer()[1] * 100
+	# 1 millibar = 100 Pascal
+	return sensor_tag_collector.get_barometer()[1] * 100
 
 
 def get_light_level(sensor_tag_collector):
-    return sensor_tag_collector.get_light_level()
+	return sensor_tag_collector.get_light_level()
 
 
 def get_vibration_level(sensor_tag_collector):
-    # Accelerometer x,y,z in g
-    x, y, z = sensor_tag_collector.get_accelerometer()
-    # Magnitude of acceleration
-    # ∣a⃗∣=√ (x*x+y*y+z*z)
-    vib = math.sqrt((x * x + y * y + z * z))
-    return vib
+	# Accelerometer x,y,z in g
+	x, y, z = sensor_tag_collector.get_accelerometer()
+	# Magnitude of acceleration
+	# ∣a⃗∣=√ (x*x+y*y+z*z)
+	vib = math.sqrt((x * x + y * y + z * z))
+	return vib
 
 
 def get_rpm(sensor_tag_collector):
-    # RPM of Z-axis
-    # Average of 5 samples
-    _rpm_list = []
-    while True:
-        if len(_rpm_list) == 5:
-            rpm = 0
-            for _ in _rpm_list:
-                rpm += _
-            rpm = int(rpm / 5)
-            break
-        else:
-            z_degree = sensor_tag_collector.get_gyroscope()[2]
-            # (°/s to RPM)
-            _rpm_list.append(int((abs(z_degree) * 0.16667)))
-    rpm_model_queue.put(rpm)
-    return rpm
+	# RPM of Z-axis
+	# Average of 5 samples
+	_rpm_list = []
+	while True:
+		if len(_rpm_list) == 5:
+			rpm = 0
+			for _ in _rpm_list:
+				rpm += _
+			rpm = int(rpm / 5)
+			break
+		else:
+			z_degree = sensor_tag_collector.get_gyroscope()[2]
+			# (°/s to RPM)
+			_rpm_list.append(int((abs(z_degree) * 0.16667)))
+	rpm_model_queue.put(rpm)
+	return rpm
 
 
 def get_rpm_for_model():
-    return rpm_model_queue.get(block=True)
+	return rpm_model_queue.get(block=True)
 
 
 def action_actuator(value):
-    print value
+	print value
```

Comment:
actuator_udm can be used to pass on the value to the actuator, as of now we are printing them